src/_sass/_related.scss

```diff
@@ -31,14 +31,73 @@
 
     li {
       margin: 0;
-      padding-right: 2em;
+      padding: 0.25em 3em 0.25em 0;
       white-space: nowrap;
     }
   }
 
   nav > ul {
     display: flex;
   }
+
+  .related__section-link {
+    position: relative;
+    label,
+    a {
+      display: inline;
+    }
+  }
+
+  .related__section-list {
+    display: none;
+    margin-left: 1em;
+  }
+
+  .related__expand-toggle {
+    -webkit-appearance: none;
+    appearance: none;
+    background-color: transparent;
+    margin: 0;
+  }
+
+  .related__expand-toggle:checked ~ .related__section-list {
+    display: block;
+  }
+
+  .arrow {
+    border: solid colours.$accent-b;
```

Comment:
I think the arrow colour needs better contrast, the text colour would be ok to use here but also check the focus state is fine

Reply:
Yea, there won't be any differentiation on hover if we use the text colour. Would you be able to provide a suggestion for hover? 

Focus has the default browser style: 

![image](https://github.com/dxw/accessibility-manual/assets/50752284/8ab0f92e-9f7c-48bd-a3bd-7ec0b10702b3)

Reply:
For hover the border width could increase on the arrow, e.g. 0.35em